content/en/blog/2022/v1.0-trio.md

```diff
@@ -0,0 +1,12 @@
+---
+title: OpenTelemetry Erlang/Elixir, Javascript, and Ruby v1.0
+linkTitle: Erlang/Elixir, JS, and Ruby v1.0
+date: 2022-02-01
+---
```

Comment:
@tedsuo @austinlparker : note that I chose to exclude an `author` field because this isn't really a post but rather a reference to a post. WDYT?

If you prefer, I can add the authors like this:

```yaml
spelling: cSpell:ignore Dyla Laurin Sloughter
author: Tristan Sloughter, Daniel Dyla, and Robert Laurin
```

Reply:
Yeah, this makes sense.